L1Trigger/L1CaloTrigger/interface/Phase2L1CaloPFClusterEmulator.h

```diff
@@ -71,6 +86,22 @@ namespace gctpf {
     return bestAll;
   }
 
+  inline GCTint_t getPeakOfHFStrip(const GCTEtaHFStrip_t& etaStrip) {
+    GCTint_t best01 = bestOf2(etaStrip.t[0], etaStrip.t[1]);
+    GCTint_t best23 = bestOf2(etaStrip.t[2], etaStrip.t[3]);
+    GCTint_t best45 = bestOf2(etaStrip.t[4], etaStrip.t[5]);
+    GCTint_t best67 = bestOf2(etaStrip.t[6], etaStrip.t[7]);
+    GCTint_t best89 = bestOf2(etaStrip.t[8], etaStrip.t[9]);
+    GCTint_t best1011 = bestOf2(etaStrip.t[10], etaStrip.t[11]);
+    GCTint_t best0123 = bestOf2(best01, best23);
+    GCTint_t best4567 = bestOf2(best45, best67);
+    GCTint_t best891011 = bestOf2(best89, best1011);
+    GCTint_t best01234567 = bestOf2(best0123, best4567);
+    GCTint_t bestAll = bestOf2(best01234567, best891011);
```

Comment:
this has been brought up before and to reiterate the answer: since this is a firmware based emulator we would like to keep it as is for reference in future development